index.html

```diff
@@ -231,13 +224,6 @@
       manufacturer. They may also expose an <a>NDEF</a> message.
     </p>
   </div>
-  <p>
```

Comment:
Maybe place a note that peer to peer support has been removed.

Reply:
Done.